cli/src/main/java/com/liferay/blade/cli/command/CreateArgs.java

```diff
@@ -190,7 +190,7 @@ public void setViewType(String viewType) {
 
 	@Parameter(
 		description = "If a new jsp hook fragment needs to be created, provide the name of the host bundle symbolic name. Required for \"-t fragment\".",
-		names = {"-h", "--host-bundle-bsn"}
+		names = {"-h", "--host-bundle-bsn", "--host-bundle-symbolic-name"}
```

Comment:
@gamerson , Should we remove these template specific parameters from `CreateArgs` and defer to `ProjectTemplatesFragmentArgs` etc,